src/ipvs/ip_vs_laddr.c

```diff
@@ -198,7 +198,8 @@ int dp_vs_laddr_bind(struct dp_vs_conn *conn, struct dp_vs_service *svc)
         ssin.sin_family = svc->af;
         ssin.sin_addr = laddr->addr.in;
 
-        if (sa_fetch(laddr->iface, &dsin, &ssin) != EDPVS_OK) {
+        if (sa_fetch(svc->af, laddr->iface, (struct sockaddr_storage *)&dsin,
```

Comment:
I think laddr(dp_vs_laddr) has af is better. Because ipvsadm and keepalived will judge ipv4/ipv6.

Reply:
ok